rever.xsh

```diff
@@ -4,23 +4,26 @@ $ACTIVITIES = [
     "tag",
     "push_tag",
     "ghrelease",
-    ]
+]
 
 #
 # Basic settings
 #
 $PROJECT = $GITHUB_REPO = "conda-build"
 $GITHUB_ORG = "conda"
-$AUTHORS_FILENAME = "AUTHORS.txt"
+$AUTHORS_FILENAME = "AUTHORS.rst"
 
 #
 # Changelog settings
 #
-$CHANGELOG_FILENAME = "CHANGELOG.txt"
-$CHANGELOG_PATTERN = "# current developments"
-$CHANGELOG_HEADER = """# current developments
-$RELEASE_DATE $VERSION:
-------------------
+$CHANGELOG_FILENAME = "CHANGELOG.rst"
+$CHANGELOG_PATTERN = ".. current developments"
+$CHANGELOG_HEADER = """.. current developments
+
+.. _release-$VERSION:
```

Comment:
This adds a header target so that links in the sidebar aren't generated with anonymous link fragments like `#id65` or similar. The link fragment instead will be `#release-3-1-26`. Not ideal but easier to read and most importantly unique.